samples/System.Numerics.Tensors.Samples/TensorSamples/WrapNativeMemory.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace TensorSamples
+{
+    partial class WrapNativeMemory
+    {
+        public static void RunSample()
+        {
+            Console.WriteLine($"*** {nameof(WrapNativeMemory)} ***");
+            var multTable = GetMultiplicationTable(5);
```

Comment:
I think it would be better (idiomatice sample) to allocate NamtiveMemory explicitly, then wrap it in tensor, use the tensor, and then dispose the memory, as opposed to relying on the GC to finalize the memory, e.g.:
```c#
using(var memory = AllocateNative(...)){
    var tensor = new DenseTensor<double>(...);
    ...
}
```

Reply:
The point of this sample is that its supposed to represent the case where the .net app does not know what size will be needed for the native memory and the cost of computing that is equivalent to doing the work, so it cannot be allocated by the caller.  Granted in the case in the sample is pretty trivial, but imagine a case where the tensor might be constructed as a result of analyzing a block of text or some other non-trivial bit of quantification.

I agree that ideally folks would dispose of the memory explicitly, but we also don't want to force an application to treat a tensor as disposable.  In these cases the managed footprint of the tensor itself will be small so I wouldn't expect it to get on LOH.

I do have a better sample in the test code that allocates the native memory itself and then disposes it.  I could add another scenario for that.

Reply:
I added the scenario for a pre-allocated tensor wrapping native memory.